plugins/quickinspector/quickinspector.cpp

```diff
@@ -387,6 +389,28 @@ void QuickInspector::objectSelected(void *object, const QString &typeName)
         selectSGNode(reinterpret_cast<QSGNode *>(object));
 }
 
+void QuickInspector::objectCreated(QObject *object)
+{
+    if (!object)
+        return;
+
+    if (QQuickWindow *window = qobject_cast<QQuickWindow *>(object)) {
+        if (QQuickView *view = qobject_cast<QQuickView *>(object)) {
+            m_probe->discoverObject(view->engine());
+        }
+        else {
+            QQmlContext *context = QQmlEngine::contextForObject(window);
+            QQmlEngine *engine = context ? context->engine() : Q_NULLPTR;
+
+            if (!engine) {
+                engine = qmlEngine(window->contentItem()->childItems().value(0));
```

Comment:
no, value() is compatible with out of bounds index and would simply return null.